php/class-coauthors-wp-list-table.php

```diff
@@ -10,6 +10,8 @@
 class CoAuthors_WP_List_Table extends WP_List_Table {
 
 	var $is_search = false;
+    var $filters = array();
+    var $active_filter = '';
```

Comment:
Nitpick: these lines may use tab instead of space for indentation.

Reply:
That's exactly why I created https://github.com/Automattic/Co-Authors-Plus/issues/769. Manually checking the formatting is just a waste of time IMO.